src/components/DateTimezoneComponent.tsx

```diff
@@ -29,10 +29,10 @@ export class DateTimezoneComponent extends React.Component<InlineComponentProps,
                     <Datetime
                         value={this.state.dateTime}
                         dateFormat={"LL"}
-                        viewDate={this.state.dateTime}
                         onChange={this.handleChange}
                         utc={false}
-                        inputProps={{ placeholder: 'Select ' + this.props.field.title, disabled: this.props.readonly }}
+                        timeFormat={"HH:mm"}
```

Comment:
will it always ask for time now ?

Reply:
Yeah it's a new component for date. To select time and timezone. which is need for internationalization. type: "datetimezonepicker".